raiden/transfer/state.py

```diff
@@ -186,7 +186,7 @@ def __ne__(self, other):
 
 
 class TokenNetworkState(State):
-    """ Corresponds to a channel manager smart contract. """
+    """ Corresponds to a token newtwork smart contract. """
```

Comment:
s/newtwork/network

Reply:
Token NEWTORK is what I intended to write obviously ....

Reply:
Sorry, my bad 🙊